packages/server/lib/browsers/chrome.ts

```diff
@@ -501,13 +513,16 @@ export = {
       launchOptions.preferences = _mergeChromePreferences(preferences, launchOptions.preferences as ChromePreferences)
     }
 
+    const p = _disableRestorePagesPrompt(userDir)
```

Comment:
If we keep this here, we'll want a more appropriately named variable.

Reply:
This was for debugging while running - I'll remove

Reply:
addressed in https://github.com/cypress-io/cypress/pull/29447/commits/c209969af1c93d6a5da2fcddc1f46ec0ea8b9d1d